packages/teleport/src/Kubes/__snapshots__/Kubes.story.test.tsx.snap

```diff
@@ -74,31 +118,134 @@ exports[`failed 1`] = `
     >
       Kubernetes
     </div>
+    <a
+      class="c3"
+      href="https://goteleport.com/docs/kubernetes-access/guides/multiple-clusters/"
+      kind="primary"
+      target="_blank"
+    >
+      View documentation
+    </a>
   </div>
   <div
-    class="c3"
+    class="c4"
     kind="danger"
   >
     server error
   </div>
 </div>
 `;
 
+exports[`kube event dialogue 1`] = `null`;
```

Comment:
when we snapshot, we need to check the snapshot result to manually check that it is the intended result

Reply:
API Testing library helped with how to snapshot test a portal component.